clients/rospy/src/rospy/core.py

```diff
@@ -109,7 +110,7 @@ def parse_rosrpc_uri(uri):
     @raise ParameterInvalid: if uri is not a valid ROSRPC URI
     """
     if uri.startswith(ROSRPC):
-        dest_addr = uri[len(ROSRPC):]            
+        dest_addr = uri[len(ROSRPC):]

```

Comment:
I'll fix this again, I think the file has mixed line endings and my editor is trying to be helpful